pkg/clients/scheduler/client.go

```diff
@@ -13,6 +13,7 @@ import (
 	"io"
 	"io/ioutil"
 	"net/http"
+	"time"
```

Comment:
Be advised that the GetSchedulerClient() function below is outdated and should be replaced in order to facilitate the endpoint monitor pattern. See NewDeviceClient() below for an example:
https://github.com/edgexfoundry/edgex-go/blob/master/pkg/clients/metadata/device.go#L66